spark/src/main/scala/org/apache/spark/sql/delta/schema/SchemaUtils.scala

```diff
@@ -1263,6 +1263,13 @@ def normalizeColumnNamesInDataType(
     unsupportedDataTypes.toSeq
   }
 
+  /**
+   * Find VariantType columns in the table schema.
```

Comment:
`Find` in the description and `Boolean` as the return type seems off to me.

`Returns 'true' if any XXX exists in the table schema`